doc/classes/Transform3D.xml

```diff
@@ -247,28 +247,28 @@
 			<return type="Transform3D" />
 			<param index="0" name="right" type="float" />
 			<description>
-				Multiplies all components of the [Transform3D] by the given [float], including the [member origin]. This affects the transform's scale uniformly, also resizing the [member basis].
+				Multiplies all components of the [Transform3D] by the given [float], including the [member origin]. This affects the transform's scale uniformly, scaling the [member basis].
```

Comment:
```suggestion
				Multiplies all components of the [Transform3D] by the given [float], including the [member origin]. This affects the transform's scale uniformly, and scaling the [member basis].
```

This happens around 4 times. Isn't this a list?

Reply:
It isn't a list, it's a sub clause

Reply:
Sounds good.

Reply:
I can reword the description outright if it's giving trouble. I just want to make it clear that the whole transform (`origin` **AND** `basis`) are affected.